xiilib/database_migration.py

```diff
@@ -129,26 +85,28 @@ def run(self, environment: dict[str, str], working_dir: pathlib.Path) -> None:
             DatabaseMigrationStatus.FAILED,
         ):
             return
-        if not self.script:
-            return
-        logger.info("execute database migration script: %s", repr(self.script))
+        logger.info("execute database migration command: %s", command)
         try:
-            self._container.exec(
-                ["/bin/bash", "-xeo", "pipefail", self.script],
+            stdout, stderr = self._container.exec(
+                command,
                 environment=environment,
                 working_dir=str(working_dir),
             ).wait_output()
             self._set_status(DatabaseMigrationStatus.COMPLETED)
-            self._set_completed_script(self.script)
+            logger.info(
+                "database migration command %s completed, stdout: %s, stderr: %s",
+                command,
+                stdout,
```

Comment:
The script could be very verbose, shouldn't we log stdout/stderr in `debug` ?

Reply:
Well, it only runs once for every charm revision, I am okay to put it in `info` or `debug`

Reply:
It's just that we don't control what the script outputs... some from one side it's interesting for the user to get it in debug log... from the other side a migration script could throw thousands of lines in the logs :/

Reply:
I think if the log level is `info` and the log is printed to `debug`, then it will be lost forever.

Reply:
On the otherside dropping lots of lines in juju logs will make them somehow useless. We could leave it like this but specify in the documentation that the user needs to be careful with the verbosity of the script.